dashboard/test/lib/expired_deleted_account_purger_test.rb

```diff
@@ -481,7 +493,8 @@ def teardown
       Done purging user_id #{students[5].id}
       AccountsPurged: 6
       AccountsQueued: 0
-      ManualReviewQueueDepth: #{QueuedAccountPurge.all.count}
+      ReviewQueueDepth: #{QueuedAccountPurge.count}
```

Comment:
This used to have `.all`. Why don't we want it anymore?

Reply:
We just want to count number of records in the `QueuedAccountPurge` model, using `.all` was redundant.